front/src/common/Map/const.ts

```diff
@@ -3,7 +3,7 @@ import config from 'config/config';
 export * from 'common/Map/Consts/SignalsNames';
 
 // Map constants
-export const MAP_URL = config.proxy_editoast + "/layers";
+export const MAP_URL = config.proxy_editoast + '/layers';
```

Comment:
Use string concatenation here
```suggestion
export const MAP_URL = `${config.proxy_editoast}/layers`;
```

Reply:
done